typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -827,6 +827,20 @@ function fillPlotTemplate(
     const scale =
         templateTrack?.scale !== options.scale ? options.scale : undefined;
 
+    let colorTableName = "";
+    if (typeof options.colorTable === "function") {
+        colorTableName = "Function";
+    } else if (options.colorTable) {
+        colorTableName = options.colorTable.name;
+    }
+
+    let inverseColorTableName = "";
+    if (typeof options.inverseColorTable === "function") {
+        inverseColorTableName = "Function";
+    } else if (options.inverseColorTable) {
```

Comment:
replace by:
} else {
colorTableName = options.inverseColorTable?.name ?? "";
}

Reply:
changed it to a conditional (ternary) operator:

```
        inverseColorTable:
            typeof options.inverseColorTable === "function"
                ? "Function"
                : options.inverseColorTable?.name ?? "",
```